### ISSUE TITLE
ArchMatCounter does not log any materials

### ISSUE THREAD

**Issue by Resolude** [ZeroGwafa/ArchMatCounter#25]
Issue: Excavating materials with the app in a seemingly-working state does not track any mats in the chat. I'm using a porter and a brush, but nothing in the chat is being logged, neither materials nor artifacts.

Options enabled at the time of issue: None; app is in a default state. This is my first time attempting to use it, and I did do a factory reset for good measure. In addition, transparency is disabled for the interface and timestamps are showing (with more than five timestamps visible).

Steps to reproduce: Set up the app, wait for it to detect chat successfully, follow the usage steps, and excavate materials from the Bibliotheke debris in Everlight

Am I missing something obvious? What else can I provide to help diagnose? Console log didn't have anything obvious in it.

**Comment by Resolude** [comment]
I tried the same excavation site but with the porter disabled (using a Grace of the Elves btw) and saw the same result. I don't seem to be able to make the application log any of the materials that are excavated. I also tried increasing the chat box font size from 10 to 12, quitting and re-opening the application, but no change. AfkWarden tolerates both font sizes and I'm unsure if it matters, but worth mentioning.

https://www.reddit.com/r/runescape/comments/1ge0fgu/psa_all_alt1_apps_that_require_reading_the/
^ mentions that the app has been updated since the breaking change last week, so I figure it's something on my end but I can't quite figure it out.
